packages/link/src/Link.js

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
```

Comment:
I don't think that we want to directly tie this package to react-router.  

One solution might be to have this component use a regular `a` tag, but allow the base component to be specified via a prop.  Then users could pass in a react-router `Link` or `NavLink` and that would get custom styled.  As long as all props, including the `to` prop,  are passed through to the base component, this should work. This would avoid any direct dependencies.  We would of course need some examples, demonstrating this functionality.